Source/Public/Remove-VSTeamWorkItemTag.ps1

```diff
@@ -0,0 +1,32 @@
+function Remove-VSTeamWorkItemTag {
+   [CmdletBinding(SupportsShouldProcess = $true, ConfirmImpact = "High", DefaultParameterSetName = 'ByID',
+    HelpUri='https://methodsandpractices.github.io/vsteam-docs/docs/modules/vsteam/commands/Remove-VSTeamWorkItemTag')]
+    param(
+      [Parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true)]
+      [string] $TagIdOrName,
+
+      [switch] $Force,
+
+      [Parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true)]
+      [vsteam_lib.ProjectValidateAttribute($false)]
+      [ArgumentCompleter([vsteam_lib.ProjectCompleter])]
+      [string] $ProjectName
+   )
+
+   Process {
+      if ($Force -or $pscmdlet.ShouldProcess($item, "Delete Work Item Tag")) {
```

Comment:
The message could be a bit clearer. Something like:

"Delete work Item tag from all work items in the project"